package.json

```diff
@@ -5,7 +5,7 @@
     "node": ">=8.0.0"
   },
   "author": "BitPay <dev@bitpay.com>",
```

Comment:
Not a blocker! But, food for thought. Is it time to add OIPWG as an author?

Reply:
We could switch it over to be OIPWG as the author, but since the bulk of the work was done by BitPay I had just been leaving it alone. In the future we could change it to be an array of contributors and place OIPWG in there :)